libraries/ashrae/guideline36/4.6-multiple-zone-vav-ahu.yml

```diff
@@ -0,0 +1,135 @@
+multiple-zone-ahu:
+  body: >
+    @prefix p: <urn:___param___#> .
+    @prefix brick: <https://brickschema.org/schema/Brick#> .
+    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+    p:name a brick:AHU ;
+      brick:hasPart
+        ( p:supply-fan
+          brick:hasPoint p:supply-fan-reset ),
+        p:filter,
+        ( p:htg-coil
+          brick:hasPoint p:htg-coil-sa-temp ),
+        (_:economizer rdf:type rdf:Alt
+          rdf:_1
+            p:economizer-damper
+          rdf:_2
+            p:economizer-minimum-oa-damper
+              brick:hasPoint p:economizer-minimum-oa-damper-flow
+          rdf:_3
+            p:economizer-damper
+            p:minimum-oa-damper
+              brick:hasPoint p:minimum-oa-damper-pressure
+          rdf:_4
+            p:economizer-damper
+            p:minimum-oa-damper-afms
+              brick:hasPoint p:minimum-oa-damper-afms-flow ),
+        p:return-damper,
+        p:clg-coil,
+        (_:relief rdf:type rdf:Alt
+          rdf:_1
+            p:relief-damper
+              brick:hasPoint p:bldg-static-pressure
+          rdf:_2
+            p:relief-fan
+            p:relief-fan-damper
+              brick:hasPoint p:bldg-static-pressure ),
+        ( p:return-fan
+            brick:hasPoint
+              p:return-fan-reset
+              p:return-fan-sa-flow
+              p:return-fan-ra-flow
+              p:return-fan-pressure
+              p:bldg-static-pressure
+          p:return-fan-exhaust-damper ) ;
```

Comment:
I removed the separate template in the original file, which didn't appear to work as intended in G36 because it's optional.

https://github.com/NREL/BuildingMOTIF/blob/b1924b8e954e86647bdfeff970c80f1a751c3949/libraries/ashrae/guideline36/4.6-multiple-zone-vav-AHU.yml#L52-L73